.prettierrc

```diff
@@ -1,3 +1,3 @@
 {
-  "semi": false
+  "semi": true
```

Comment:
why change this?

Reply:
I think I accidentally changed it, i will fix it